docs/UserGuide.md

```diff
@@ -87,13 +84,89 @@ commands may receive.
 </div>
 
 #### **2.2.3 Types of User Input**
-To expand.
-
+In this section, you will learn about the commonly used User Inputs as well as the prefixes that precede them. These will be helpful when you are trying to specify the fields for certain commands.
+<br><br>
 ##### `NAME`
-* The name of the customer who has placed the order.
-* The name of the customer must not be empty, it can take in numbers, letters or a mix of both.
-
-:information_source: _Fun Fact: Elon Musk, the world richest man has a son named X Æ A-12._
+The name of the customer who has placed the order.<br>
+Prefix: `n/`
+* It can contain alphabetical characters, numbers and spaces.<br>
+  E.g. `Alex Yeoh`, `Johnathan9`, `Charlotte the 5th`
+* It cannot be longer than 80 characters.
+* It cannot be empty.
+<br><br>
+##### `PHONE_NUMBER`
+The phone number of the customer who has placed the order.<br>
+Prefix: `p/`
+* It can only contain numbers.
+* It should be at least 3 digits long.
+* It cannot be longer than 20 digits.
+<br><br>
+##### `EMAIL`
+The email of the customer who has placed the order.<br>
+Prefix: `e/`
+* Emails should be of the format `local-part@domain`.
+* `local-part` can contain alphabetical or numerical characters and these special characters: ``!#$%&'*+/=?`{|}~^.-``.<br>
+  E.g. `alice#3in?wonderland!`
+* `domain` should
+  * be at least 2 characters long
+  * start and end with alphabetical or numerical characters
+  * contain alphabetical or numerical characters, a period `.` or a hyphen `-` for the characters in between, if any.<br>
+  E.g. `sample-domain.com`
+<br><br>
+##### `ADDRESS`
+The address of the customer who has placed the order.<br>
+Prefix: `a/`
+* It can contain any characters.
+* It cannot be empty.
+<br><br>
+##### `DELIVERY_DATE`
+The delivery date for the order.<br>
+Prefix: `d/`
+* It should be a valid calendar date.
+* It should be any of the format:
+  * `dd/MM/yyyy` E.g. `01/12/2021`
+  * `dd-MM-yyyy` E.g. `31-12-2021`
+  * `dd.MM.yyyy` E.g. `01.12.2021`
+  * `dd MMM yyyy` E.g. `31 Dec 2021`
+* When adding or editing an order using the commands, the `DELIVERY_DATE` should be a future date.<br>
+  I.e. the date today or a date after today.
+* `DELIVERY_DATE` can be stored as a date in the past in the application. Orders with a `DELIVERY_DATE` before today's date will not be deleted.<br>
```

Comment:
first statement can be deleted, a little confusing from the perspective of a tech noob